pkg/grpc/server.go

```diff
@@ -7,7 +7,7 @@ import (
 	configuration "github.com/buildbarn/bb-storage/pkg/proto/configuration/grpc"
 	"github.com/buildbarn/bb-storage/pkg/util"
 	"github.com/golang/protobuf/ptypes"
-	"github.com/grpc-ecosystem/go-grpc-prometheus"
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
I've seen changes along these lines in the past. Why is this being done? It's not as if our imports are ambiguous. Could you please revert this?

Reply:
The reason is specifically that goimports doesn't believe you and wants to remove it, even though it does in fact compile.

Reply:
I've fumbled these through for now, we can puzzle out goimports independently.